alerts/apps_alerts.libsonnet

```diff
@@ -208,13 +208,13 @@
           },
           {
             expr: |||
-              sum by (namespace, pod, container, %(clusterLabel)s) (kube_pod_container_status_waiting_reason{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s}) > 0
+              kube_pod_container_status_waiting_reason{reason!="CrashLoopBackOff", %(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s} > 0
```

Comment:
The addition of the label selector `reason!="CrashLoopBackOff` is a change in behaviour for this alert, can you provide some more detail around this? Presumably, it is to avoid duplicate alerts which would be covered already by `KubePodCrashLooping`?

Reply:
correct, this is due there is a dedicated CrashLoopBackOff alert already present.